packages/prepo-ui/src/components/Simulator/utils.ts

```diff
@@ -0,0 +1,10 @@
+export const getPositionFromPayoutRange = (
+  payoutRange: [number, number],
+  price: number
+): number => {
+  const [floor, ceiling] = payoutRange
+  const position = (price - floor) / (ceiling - floor)
+  if (position < 0) return 0
+  if (position > 1) return 1
+  return position
```

Comment:
```suggestion
  clamp(position, 0, 1)
```

From `lodash/clamp`. I think it's easier to read, but feel free to disregard - I can see the argument for not using lodash.

Reply:
yea I don't like lodash haha it's too heavy, will disregard this, but thanks for the feedback!